packages/gestalt/src/InternalOverlayPanel.js

```diff
@@ -25,7 +25,7 @@ type NodeOrRenderProp = Node | (({| onDismissStart: () => void |}) => Node);
 
 type InternalSheetProps = {|
   accessibilityDismissButtonLabel?: string,
-  accessibilitySheetLabel: string,
+  accessibilityOverlayPanelLabel: string,
```

Comment:
Should this just be `accessibilityLabel`? Is the differentiation of repeating the component name in the prop name necessary?

Reply:
It's internal,  could simplify but leaving as is. no biggie

Reply:
DONE